docs/user-guide/cli-using-integrating-apiml.md

```diff
@@ -40,53 +55,101 @@ Issue the following command:
 
 ```zowe auth logout```
 
-The token is expired. Log in again to obtain a new token.
+The token expires. Log in again to obtain a new token.
 
 ## Accessing a service through API ML
 
-To access services through API ML using the token in your base profile, specify the following options on commands and service profiles:
+To access mainframe services through API ML using the token in your base profile, use the following commands options:
+
+* `--base-path`: Indicates the base path of the API ML instance that you want to access.
+
+    - To establish a base path, see instructions for [Zowe V2 profiles](#specifying-a-base-path-with-zowe-v2-profiles) or [Zowe V1 profiles](#specifying-a-base-path-with-zowe-v1-profiles).
+* `--disable-defaults`: Prevents default values from being stored in service profiles. If you do not use this flag, the defaults can override values in your base profile.
+
+**Note:** Ensure that you *do not* provide username, password, host, or port directly on the service commands or profiles. Supplying those options causes the CLI to ignore the API ML token in your base profile and access the service directly.
+
+### Specifying a base path with Zowe V2 profiles
+
+Follow these steps:
+
+1. Note the complete path for a z/OSMF instance registered to API ML.
+
+    As an example, use the following path:
+
+    ```
+    https://myapilayerhost:port/ibmzosmf/api/v1
+    ```
 
-* `--base-path`: The base path of the API ML instance that you want to access.
-* `--disable-defaults`: Specify this flag to prevent default values from being stored in service profiles. If you do not use this flag, the defaults can override values in your base profile.
+    The format of base path can vary based on how API ML is configured at your site.
 
-**Note:** Ensure that you *do not* provide username, password, host, or port directly on the service commands or profiles. Supplying those options causes the CLI to ignore the API ML token in your base profile and directly access the service.
+2. To access that API ML instance, create a [service profile](../user-guide/cli-using-using-team-profiles#zowe-cli-profile-types) (or issue a command) with the `--base-path` value of `ibmzosmf/api/v1`. Your service profile uses the token and credentials stored in your default base profile.
```

Comment:
Change to:
Access the API ML instance by creating a ... (or issuing a command) ..."